docs/installation/kubernetes-installation.md

```
@@ -161,6 +161,43 @@ curl -X POST http://localhost:8080/api/chat \
+  secretName: holmes-tls               # required: secret with tls.crt and tls.key
+  # keyfilePasswordSecretKey: ""       # optional: key in the secret with the encrypted-key password
+  # caCertsSecretKey: ca.crt           # optional: a key in the same secret; enables mTLS
+```
+
+To require client certificates (**mutual TLS**), add a CA bundle under an extra key in the same secret (e.g. `ca.crt`) and set `caCertsSecretKey` to that key name. Clients without a certificate signed by that CA are rejected.
+
+> **Note**: Enabling `tls` without `secretName` fails the Helm render with a clear error, so a misconfiguration can't silently downgrade to HTTP. The server reads the certificate once at startup and does not hot-reload it — after rotating the certificate/secret, restart the pod (`kubectl rollout restart deployment/{release-name}-holmes`).
+
+After enabling, port-forward and call the service over `https` (use `-k` for a self-signed/private-CA certificate):
+
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)